movies/src/main/java/com/shotcutter/movies/TMDB/TMDBIntegrator.java

```diff
@@ -38,7 +39,7 @@
         this.movieRepository = movieRepository;
 
         pageSize = 20;
-        currentPage = ((int) movieRepository.count()) / pageSize;
+        currentPage = movieRepository.count().block().intValue() / pageSize;
```

Comment:
are you sure we cannot write it without blocking?

Reply:
I guess that it's possible, but it's a matter of refactoring, so I'd just leave a TODO for a next pr on that, thx